### PR TITLE
Add greeting for Josie

### PR DESCRIPTION
**Pull request by NicMcPhee**
This adds Josie's greeting to the Java code.

### CODE REVIEW

File: src/main/java/hellos/Hellos.java [hunk 1]
```diff
@@ -9,12 +9,17 @@ public class Hellos {
   public String generateOutput() {
     StringBuilder builder = new StringBuilder();
 
+    builder.append(josieSaysHello());
```
Comment (JosieBarber) [inline src/main/java/hellos/Hellos.java:12]: Oh my god I wrote that!
Correct ordering!

File: src/main/java/hellos/Hellos.java [hunk 2]
```diff
@@ -9,12 +9,17 @@ public class Hellos {
...
     builder.append(kkSaysHello());
     builder.append(nicSaysHello());
 
     return builder.toString();
   }
 
+  private String josieSaysHello() {
```
Comment (NicMcPhee) [inline src/main/java/hellos/Hellos.java:19]: This is a pointless comment.

**Review by JosieBarber — APPROVED** [review]

File: src/main/java/hellos/Hellos.java [hunk 3]
```diff
@@ -9,12 +9,17 @@ public class Hellos {
...
+    return "Josie says 'Howdy!'\n";
```
Comment (NicMcPhee) [inline src/main/java/hellos/Hellos.java:20]:
```suggestion
    return "Josie says 'Hello!'\n";
```

**Review by NicMcPhee — COMMENTED** [review]
This looks great - thanks!

File: src/main/java/hellos/Hellos.java (hunk 1 above)
Comment (Bonesbhbhbh) [inline src/main/java/hellos/Hellos.java:12]: great code here 10/10 work, great formatting

**Review by Bonesbhbhbh — APPROVED** [review]

File: src/main/java/hellos/Hellos.java (hunk 3 above)
Comment (NicMcPhee) [reply src/main/java/hellos/Hellos.java:20]: That's a dumb idea - leave my greeting alone.

**Pull request merged**